test/pkg/environment/common/setup.go

```diff
@@ -321,3 +317,7 @@ func (env *Environment) dumpNodeEvents(testStartTime time.Time) {
 		fmt.Print(getEventInformation("node", k, v))
 	}
 }
+
+func containsObjectType(objs []client.Object, obj client.Object) bool {
```

Comment:
optional: I typically making one line wrappers for higher order functions unless there's some real differentiating magic going on.